scenarios/waymo_motion/waymo_utility.py

```diff
@@ -0,0 +1,2226 @@
+                (point,) = plt.plot(xs[0], ys[0], "r*")
+                handles.append(
+                    Line2D(
+                        [],
+                        [],
+                        color="red",
+                        marker="*",
+                        linestyle="None",
+                        markersize=5,
+                        label=f"Cyclist {k}",
+                    )
+                )
+            elif int(k) in objects_of_interest:
+                (point,) = plt.plot(xs[0], ys[0], "g*")
+                handles.append(
+                    Line2D(
+                        [],
+                        [],
+                        color="green",
+                        marker="*",
+                        linestyle="None",
+                        markersize=5,
+                        label=f"Interactive Cyclist {k}",
+                    )
+                )
+            else:
+                (point,) = plt.plot(xs[0], ys[0], "y*")
+        else:
+            if str(k) in track_ids:
+                (point,) = plt.plot(xs[0], ys[0], "r8")
+                handles.append(
+                    Line2D(
+                        [],
+                        [],
+                        color="red",
+                        marker="8",
+                        linestyle="None",
+                        markersize=5,
+                        label=f"Other {k}",
+                    )
+                )
+            elif int(k) in objects_of_interest:
+                (point,) = plt.plot(xs[0], ys[0], "g8")
+                handles.append(
+                    Line2D(
+                        [],
+                        [],
+                        color="green",
+                        marker="8",
+                        linestyle="None",
+                        markersize=5,
+                        label=f"Interactive Other {k}",
+                    )
+                )
+            else:
+                (point,) = plt.plot(xs[0], ys[0], "k8")
+        data.append((xs, ys))
+        points.append(point)
+
+    return data, points, max_len, handles
+
+
+def plot_scenarios(
+    scenario_infos, animate: bool, feature_ids: Optional[List[str]] = None
+):
+    def plot_map_and_get_animate(scenario_info, animate_trajectories, fig_num, f_ids):
+        anim = None
+        # Get map feature data from map proto
+        map_features = scenario_info[1]
+
+        # Plot map
+        fig = plt.figure(num=fig_num)
+        if animate_trajectories or not f_ids:
+            highlighted_handles = plot_map_features(map_features, [])
+        else:
+            highlighted_handles = plot_map_features(map_features, f_ids)
+        plt.title(f"Scenario {scenario_info[0].scenario_id}")
+
+        # Set Legend Handles
+        all_handles = get_map_handles()
+        all_handles.extend(highlighted_handles)
+
+        # Resize figure
+        mng = plt.get_current_fig_manager()
+        mng.resize(1000, 1000)
```

Comment:
This was from my original script, I don't think we should resize to a hardcoded resolution.